qt/python/mantidqtinterfaces/mantidqtinterfaces/Muon/GUI/Common/contexts/fitting_contexts/model_fitting_context.py

```diff
@@ -6,6 +6,17 @@
 # SPDX - License - Identifier: GPL - 3.0 +
 from mantidqtinterfaces.Muon.GUI.Common.contexts.fitting_contexts.basic_fitting_context import BasicFittingContext
 
+PARAMETERUNITS = {
+    'Frequency': ['MHz', 'MHz'],
+    'Lambda': [r'\mu s^{-1}', r'$\mu$ $s^{-1}$'],
+    'Phi': ['Radians', 'Radians'],
+    'sample_magn_field': ['G', 'G'],
+    'sample_temp': ['K', 'K'],
+    'Sigma': [r'\mu s^{-1}', r'$\mu$ $s^{-1}$'],
+    'test1': ['test_unit_1', 'test_unit_1'],
```

Comment:
do we want to keep the test values and we should add a comment explaining why there are two versions of all the units

Reply:
Sure, I put them in for unit tests but I can use any existing unit for that

Reply:
the over option is to remove them here and then replace the dict in the unit test with one that includes these